### DISCUSSION TITLE
折りたたみにマウスカーソルをホバーしてもカーソルのスタイルが変わらない

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#289] [要望・アイデア・不具合]
`<details>` タグと `<summary>` タグを利用して折りたたみを作成することが出来ますが、ここにマウスカーソルをホバーしてもスタイルが変わりません。このため、折りたたみを知らない人が読んだ際、折りたたみに気付けない可能性があります。

折りたたみをホバーした際にも、マウスカーソルが以下の赤丸ような指 :point_up_2: になると良いと思います。

![qiitaMouseSymbol](https://user-images.githubusercontent.com/75878492/187739056-a4c42de7-a140-411e-9d08-ac9a0d7b990a.png)

### 現在のマウスカーソルの様子

![qiitaDetails](https://user-images.githubusercontent.com/75878492/187739060-22f01819-971e-418e-86c5-2ba8dd9ba93c.gif)

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ありがとうございます🙏

こちら対応させていただきます！反映までお待ちいただけますと幸いです🙇　https://github.com/increments/qiita-discussions/issues/290

**Comment by wataru86** [comment]
@Yarakashi-Kikohshi
大変お待たせいたしました :bow:
折りたたみ要素にホバーした際のカーソルのスタイルを修正しました！

https://user-images.githubusercontent.com/34445284/192700614-9673a77c-a644-4d35-ae30-e117579ee003.mov

**Reply by yarakos95** [reply]
対応ありがとうございます :smile:
見やすくなりました！！
